Projects/My_website/myWebsite/blog/views.py

```diff
@@ -26,16 +26,58 @@ class userBlog(APIView):
     authentication_classes = [TokenAuthentication]
     permission_classes = [IsAuthenticated]
 
+    def get_item(self,slug):
+        item = Post.objects.get(slug=slug)
```

Comment:
@codal-uchoudhary 
What if there's no data that that matches the slug name ? Won't the `GET` query throw an error.
Make necessary changes.

Reply:
> @codal-uchoudhary What if there's no data that that matches the slug name ? Won't the `GET` query throw an error. Make necessary changes.

Updated

Reply:
> @codal-uchoudhary What if there's no data that that matches the slug name ? Won't the `GET` query throw an error. Make necessary changes.

![Uploading Screenshot 2024-02-26 114400.png…]()

Reply:
<img width="651" alt="Screenshot 2024-02-26 114400" src="https://github.com/codal-uchoudhary/python-training-2024/assets/156191290/29153b1c-1c42-4b55-a886-c065a01a5018">

Reply:
@codal-uchoudhary 
In the delete request, why are we passing `Money-uhdbfu-dsfjn` ? What & why is that ? To delete a blog, don't 
you think we must be passing ID as the unique identifier ?

Reply:
@arvish-codal  , my intension is to pass wrong slug to check error , we get invalid slug in the response

Reply:
@codal-uchoudhary , in that case. Pass any random blog ID..i.e the one which doesn't exist in the DB at all.

**_

> Unique identified for a blog should always be it's ID...no any other field.

_**